cl/transition/impl/eth2/operations.go

```diff
@@ -820,7 +963,9 @@ func (I *impl) ProcessEth1Data(state abstract.BeaconState, eth1Data *cltypes.Eth
 		return true
 	})
 
-	if uint64(numVotes*2) > state.BeaconConfig().EpochsPerEth1VotingPeriod*state.BeaconConfig().SlotsPerEpoch {
+	if uint64(
+		numVotes*2,
```

Comment:
hmm this mysterious format shows formatter is now always perfect
nothing but just murmur

Reply:
yes i see, will increase lines